frontend/appflowy_flutter/lib/plugins/database_view/application/tar_bar_bloc.dart

```diff
@@ -64,7 +64,7 @@ class GridTabBarBloc extends Bloc<GridTabBarEvent, GridTabBarState> {
             if (updatePB.createChildViews.isNotEmpty) {
               final allTabBars = [
                 ...state.tabBars,
-                ...updatePB.createChildViews.map((e) => TarBar(view: e))
+                ...updatePB.createChildViews.map((e) => TarBar(view: e)),
```

Comment:
Is `TarBar` named correctly?

Reply:
No, but tt's not related to this PR. Let me fix it in the next PR.